### DISCUSSION TITLE
FDS output quantities : PLOT3D (.xyz and .q files)

### DISCUSSION THREAD

**Discussion by Sagar-Singhal** [firemodels/fds#13851] [Q&A]
Hi, 

I was simulating wind-blown pool fire using FDS. The simulation was successful. I extracted PLOT3D files at different time instances. I was also able to export it to Paraview for visualization of different quantities at multiple planes. But now I want to export data (especially looking for extracting absolute values of X, Y, Z velocity 2D field at multiple planes) into Python or MATLAB for further processing.  I want to perform spectral POD on 2D velocity field data. But I am unable to get data. I also tried using fds2ascii to extract data.
[example.txt](https://github.com/user-attachments/files/18036715/example.txt)

**Comment by Er9y714** [comment]
Do these two discussions help? #12822 #13804
